docs/Earn/liquid-staking/dot/stake-dot.md

```diff
@@ -82,4 +103,12 @@ Click to allow access.
 
 Successful access to aDOTb Tokens
```

Comment:
This enables access to aDOTb Tokens.
Successful addition to your wallet is indicated by .....

Reply:
thx